src/gui/accountmanager.cpp

```diff
@@ -435,13 +436,15 @@ QString AccountManager::generateFreeAccountId() const
     }
 }
 
-void AccountManager::addAccountState(AccountState *accountState)
+void AccountManager::addAccountState(AccountStatePtr accountState)
 {
+    OC_ASSERT(accountState != nullptr)
```

Comment:
Q_ASSERT(accountState) ?
I mean this should really not happen, and if it does we would need to handle it.

Reply:
No, it cannot happen.